App/server/routes/scheduled_actuator.py

```diff
@@ -0,0 +1,93 @@
+import sys
+from typing import List
+
+
+from fastapi import APIRouter, Body, HTTPException, Request, status
+
+# delete import of Response ^^^
+from fastapi.encoders import jsonable_encoder
+
+try:
+    from App.server.models.scheduled_actuator import Scheduled_Actuator, SA_Update
+except ModuleNotFoundError:
+    sys.path.append("../server")
+    from server.models.scheduled_actuator import Scheduled_Actuator, SA_Update
+
+router = APIRouter()
+
+
+@router.post(
+    "/",
+    response_description="Create a new scheduled actuator",
+    status_code=status.HTTP_201_CREATED,
+    response_model=Scheduled_Actuator,
+)
+def create_scheduled_actuator(
+    request: Request, scheduled_actuator: Scheduled_Actuator = Body(...)
+):
+    sa = jsonable_encoder(scheduled_actuator)
+    new_sa = request.app.database["scheduled actuator"].insert_one(sa)
+    created_sa = request.app.database["scheduled actuators"].find_one(
+        {"_id": new_sa.inserted_id}
+    )
+
+    return created_sa
+
+
+@router.get(
+    "/",
+    response_description="List scheduled actuators",
+    response_model=List[Scheduled_Actuator],
+)
+def list_scheduled_actuators(request: Request, limit: int = 1000):
+    scheduled_actuators = list(request.app.database["scheduled actuators"].find())
+    scheduled_actuators.sort(key=lambda r: r["updated_at"], reverse=True)
+
+    return scheduled_actuators[:limit]
+
+
+@router.get(
+    "/{id}",
+    response_description="Get a single scheduled actuator by id",
+    response_model=Scheduled_Actuator,
+)
+def find_scheduled_actuator(id: str, request: Request):
+    if (
+        sa := request.app.database["scheduled_actuator"].find_one({"_id": id})
```

Comment:
database["scheduled_actuator"] should be database["scheduled actuator"]. It might be helpful to make this a global variable at the top of the file and reference it in all other cases that use it.

Reply:
I'm going to update everything to reference that global variable. I am unsure if there should be separate instances of <scheduled actuator(s)>.

Like:  "Scheduled Actuator", "scheduled actuator", and "Scheduled Actuators" (the database?).